OpenEdXMobile/res/layout/row_course_outline_list.xml

```diff
@@ -125,27 +126,18 @@
             android:background="@android:color/transparent"
             android:gravity="center"
             android:paddingEnd="@dimen/widget_margin_half"
-            android:visibility="gone">
+            android:visibility="gone"
+            tools:ignore="RtlSymmetry">
 
             <FrameLayout
                 android:layout_width="@dimen/download_background_size"
                 android:layout_height="@dimen/download_background_size">
 
-                <View
-                    android:id="@+id/download_background"
-                    android:layout_width="wrap_content"
-                    android:layout_height="wrap_content"
-                    android:background="@drawable/ic_holo_background"
-                    android:visibility="gone"
-                    tools:visibility="visible" />
-
                 <androidx.appcompat.widget.AppCompatImageView
                     android:id="@+id/bulk_download"
-                    android:layout_width="@dimen/ic_large"
-                    android:layout_height="@dimen/ic_large"
+                    android:layout_width="30dp"
+                    android:layout_height="30dp"
```

Comment:
use defined dimens instated of e.g `download_background_size`. OR can create a new one.

OR can be a match parent cuz the parent already set the fixed dimen.